lighthouse-core/report/v2/report-styles.css

```diff
@@ -384,10 +385,15 @@ span.lh-node:hover {
 
 /* Perf Timeline */
 
+.lh-timeline-container {
+  overflow: hidden;
+  border-top: 1px solid var(--metric-timeline-rule-color);
+}
+
 .lh-timeline {
   padding: 0;
   padding-bottom: 0;
-  min-width: calc(var(--lh-filmstrip-thumbnail-width) * 10 + var(--default-padding) * 2);
```

Comment:
going to leave alone the empty space on the right for now. After I add in the button to view trace i'll revisit this.